plutus-use-cases/src/Plutus/Contracts/GameStateMachineWithSecretArguments.hs

```diff
@@ -0,0 +1,226 @@
+{-# LANGUAGE DataKinds             #-}
```

Comment:
TBH I'd rather just modify the existing example, it's a simple enough addition and I don't think it's worth duplicating the entire thing again.

Reply:
I agree - will fix this.